packages/app/src/specs/AverageDuration.vue

```diff
@@ -1,11 +1,17 @@
 <template>
   <div
-    v-if="props.gql.data?.__typename === 'CloudProjectSpec' && props.gql.data.averageDuration"
+    v-if="props.gql?.data?.__typename === 'CloudProjectSpec' && props.gql?.data?.averageDuration"
     class="h-full grid text-gray-700 justify-end items-center"
     data-cy="average-duration"
   >
     {{ getDurationString(props.gql.data.averageDuration) }}
   </div>
+  <div
+    v-else
+    class="h-full grid text-gray-400 justify-end items-center"
+  >
+    --
+  </div>
```

Comment:
I see. This will break some tests. I can merge back from base and take care of fixing tests.

Reply:
I'm trying to figure out what failures are directly related to these changes - all of them? Is some of it flake?